src/main/kotlin/frc/robot/subsystems/conveyor/ConveyorIOReal.kt

```diff
@@ -30,24 +56,21 @@ class ConveyorIOReal : ConveyorIO {
         roller.stopMotor()
     }
 
+    override fun setGains(kP: Double, kI: Double, kD: Double, kS: Double, kV: Double, kA: Double) {
+        roller.configurator.apply(
```

Comment:
`apply`
I also believe you can create one function that applies configurations to Talon's and use it for multiple subsystems.

Reply:
I don't think creating a function like that is really necessary.